ngraph/core/include/ngraph/op/power.hpp

```diff
@@ -108,15 +60,14 @@ namespace ngraph
                       const AutoBroadcastSpec& auto_broadcast =
                           AutoBroadcastSpec(AutoBroadcastType::NUMPY));
 
+                void validate_and_infer_types() override;
                 virtual std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
                 bool evaluate(const HostTensorVector& outputs,
                               const HostTensorVector& inputs) const override;
             };
         } // namespace v1
 
-        NGRAPH_SUPPRESS_DEPRECATED_START
-        using v0::Power;
-        NGRAPH_SUPPRESS_DEPRECATED_END
+        using v1::Power;
```

Comment:
I propose to remove this using. Because these using were introduced for nGraph backward compatibility. At the current moment you can work with Power op using opset `opsetX::Power` or version `v1::Power`

Reply:
It is required in few places (like interpreter) and I can see other ops follow the same pattern. I'd prefer to not change this right now.

Reply:
It looks like very simple change. Why we cannot do it now?
In general if we remove v0 Power we already change the behaviour.